platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -119,6 +119,17 @@ MGL_EXPORT
  */
 @property (nonatomic) CGFloat scale;
 
+/**
```

Comment:
This declaration is not necessary. As long as we have a private interface that declares the property it can be used.

Reply:
If we decide to keep this declaration then lets remove the documentation.